modules/ROOT/pages/api-groups-sla-tiers-faqs.adoc

```diff
@@ -0,0 +1,29 @@
+= SLA Tiers For API Groups FAQs
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+*Question: What happens if a product is published without any SLA tiers?*
+*Answer:*
++
+As a Group Administrator, you must carefully decide whether an API Group must not have any SLA tiers. No SLA implies unlimited SLA Tiers for all the instances in the product that do not have an SLA based policy applied. 
+
+API Group instances with an SLA based policy will not be available for consumption until they have at least one tier defined. Group Administrators must decide to create SLA tiers after creating API Groups and after publishing the API Group instances to Exchange. Otherwise, they should give a warning, such as `There are no SLA tiers defined on this API Group`.
```

Comment:
I don't understand the second sentence. we should recommend the user to create the tiers before publishing the group instance to Exchange, so when it is published it is ready to be consumed

Reply:
Done! Thanks!